app/views/shared/check_answers/_client_details.html.erb

```diff
@@ -58,6 +58,16 @@
         ) %>
   <% end %>
 
+  <% if :employment_status.in?(attributes) && !applicant.employed.nil? %>
+    <%= check_answer_link(
+          name: :employment_status,
+          url: nil,
```

Comment:
Should there be a link to the applicant employment status view?

Reply:
I don't think so no. There aren't links to any of the other client details on this page and I believe it would be too late in the journey for the user to change the employment status. However, I have removed url: nil for clarity.